src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoFPHSExternalIdentifierDaoTest.java

```diff
@@ -0,0 +1,176 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.sagebionetworks.bridge.TestConstants.TEST_STUDY_IDENTIFIER;
+import static org.sagebionetworks.bridge.dynamodb.DynamoFPHSExternalIdentifierDao.CONFIG_KEY_ADD_LIMIT;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.util.List;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBScanExpression;
+import com.amazonaws.services.dynamodbv2.datamodeling.PaginatedScanList;
+import com.google.common.collect.ImmutableList;
+
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import org.sagebionetworks.bridge.config.BridgeConfig;
+import org.sagebionetworks.bridge.exceptions.BadRequestException;
+import org.sagebionetworks.bridge.exceptions.EntityAlreadyExistsException;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.accounts.FPHSExternalIdentifier;
+
+public class DynamoFPHSExternalIdentifierDaoTest extends Mockito {
```

Comment:
Missing tests for

* verify an already registered external ID
* registering a non-existent external ID

Reply:
Added.